eng/common/pipelines/templates/steps/update-docsms-metadata.yml

```diff
@@ -26,8 +26,19 @@ parameters:
     type: object
     default:
       - '**'
-
+  - name: RetrieveCodeOwnersToolVersion
+    type: string
+    default: '1.0.0-dev.20211108.8'
+  - name: DevOpsFeed
+    type: string
+    default: "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json"
 steps:
+- pwsh: |
+    Write-Host "$(RetrieveCodeOwnerToolVersion)"
+    Write-Host "$(RetrieveCodeOwnerToolName)"
```

Comment:
I would add more context to these log messages about what values you're printing, as it won't be necessarily be clear to someone reading them in the pipeline.

Reply:
Also what is the difference between the RetrieveCodeOwner*s*ToolVersion parameter and the RetrieveCodeOwnerToolVersion variable?

Reply:
`RetrieveCodeOwnerToolName` also doesn't seem to exist anywhere in the repo?